File: SuperiorPackGroup/Reports/PalletTransactionXtraReport.vb
Public Class PalletTransactionXtraReport

    Private Sub xRLOverShippedTotal_BeforePrint(ByVal sender As System.Object, ByVal e As Printing.PrintEventArgs) Handles xRLOverShippedTotal.BeforePrint

        xRLOverShippedTotal.Text = (CDbl(shippedTotalXrLabel.Summary.GetResult) - CDbl(receivedTotalXRLabel.Summary.GetResult)).ToString

    End Sub

End Class

File: SuperiorPackGroup/Reports/PalletTransactionXtraReport.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace SuperiorPackGroup
{
	public partial class PalletTransactionXtraReport
	{

		public PalletTransactionXtraReport()
		{
			InitializeComponent();
		}

		private void xRLOverShippedTotal_BeforePrint(System.Object sender, System.Drawing.Printing.PrintEventArgs e)
		{

			xRLOverShippedTotal.Text = (Convert.ToDouble(shippedTotalXrLabel.Summary.GetResult()) - Convert.ToDouble(receivedTotalXRLabel.Summary.GetResult())).ToString();

		}

	}
}